### ISSUE TITLE
Create clusters for training and validation as well as for testing set

### ISSUE THREAD

**Issue by gcroci2** [DeepRank/3D-Vac#146]
We should think about how to create better sets for training and validation, since it's evident that right now the optimized validation loss is not working properly for the testing set, and thus the networks do not really generalize, but overfits on the training data. 
We should do this for
- [ ] Peptides clusters
- [ ] Alleles clusters

We can start here a discussion about it. Which are the issues on doing so right now?

**Comment by DarioMarzella** [comment]
Peptide clustering was scraped from the project. Allele clustering validation set could still be implemented for future experiment, but it's not planned at the moment.
